python-menus-toolbars/README.md

```diff
@@ -0,0 +1,13 @@
+# Sample PyQt Application
+
+A sample GUI application that shows how to create and use menus, toolbars, and status bars using Python and PyQt.
+
```

Comment:
@lpozo I'd include a short instruction for the impatient reader showing how to run this app:

```
$ pip install pyqt5
$ python sample-app.py
```

Reply:
Done!